Amplify/Categories/DataStore/DataStoreCategory+HubPayloadEventName.swift

```diff
@@ -23,4 +23,22 @@ public extension HubPayload.EventName.DataStore {
     /// Dispatched when DataStore receives a sync response from the remote API via the API category. The Hub Payload
     /// will be a `MutationEvent` instance that caused the conditional save failed.
     static let conditionalSaveFailed = "DataStore.conditionalSaveFailed"
+
+    static let networkStatusChanged = "DataStore.networkStatusChanged"
+
+    static let outboxStatusChanged = "DataStore.outboxStatusChanged"
+
+    static let subscriptionEstablished = "DataStore.subscriptionEstablished"
+
+    static let syncQueriesStarted = "DataStore.syncQueriesStarted"
+
+    static let modelSynced = "DataStore.modelSynced"
+
+    static let syncQueriesReady = "DataStore.syncQueriesReady"
+
+    static let ready = "DataStore.ready"
+
+    static let outboxMutationEnqueued = "DataStore.outboxMutationEnqueued"
+
+    static let outboxMutationProcessed = "DataStore.outboxMutationProcessed"
```

Comment:
1. Please add doc comments (like above on `conditionalSaveFailed`) describing the conditions under which these events are emitted, so a customer will know how to use them
2. I have concerns about the naming of these events
    1. Can you confirm that these are the same events as are emitted by Android & JS? I'm particularly concerned about `syncQueriesReady`--does that mean "the sync queries are ready to be executed" or "the sync queries have finished executing"?
    2. Can you confirm that the events are named the same (within platform conventions) as Android & JS?

Reply:
1. Will add comments for them.

2. Rafael J and I had the same concerns on naming. 
Say, `syncQueriesReady ` would better be `syncQueriesCompleted` or `syncQueriesFinished`.
But I have synced up with Manuel 2 weeks ago and Rafael J yesterday. The naming are the same for 3 platform.